README.md

```
@@ -90,6 +90,61 @@ Migrations live in `supabase/migrations/`; the first one enables the `postgis` e
+   | Field | Value |
+   |---|---|
+   | Host | `smtp.resend.com` |
+   | Port | `587` |
+   | Username | `resend` |
+   | Password | *(your Resend API key)* |
+   | Sender name | `Hotelyn` |
+   | Sender email | your verified address, or `onboarding@resend.dev` for staging |
+
+5. Add the two variables to your CI secrets / Supabase project secrets (see
+   `backend/.env.example`):
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)